### DISCUSSION TITLE
inquiry about get_texttrace() function

### DISCUSSION THREAD

**Discussion by yousifsamir0** [pymupdf/PyMuPDF#3105] [Looking for help]
Hello ,
I was trying to convert all text to grayscale space so and i found this function has color information,
is there any way to modify these information and update the file with it ?

**Comment by JorjMcKie** [comment]
No there is no update facility contained here.
Only PDF are supported for output with PyMuPDF, and all text extraction and searches are supported for any of the many supported document types.
